src/main/java/org/jenkinsci/plugins/p4/changes/P4ChangeSet.java

```diff
@@ -67,7 +67,7 @@ public static void store(File file, List<P4ChangeEntry> changes) {
 					stream.println("\t\t<clientId>" + cl.getClientId() + "</clientId>");
 
 					stream.println("\t\t<msg>" + P4Escaper.filter().translate(StringEscapeUtils.escapeXml(cl.getMsg())) + "</msg>");
-					stream.println("\t\t<changeUser>" + StringEscapeUtils.escapeXml(cl.getAuthor().getDisplayName())
```

Comment:
Storing the author's displayName when serializing the changeset to .xml forces the slow user lookup when parsing the changeset later. There is no advantage to storing the displayName over the author's id.